plugins/content/transcript/lib/api/filters/KalturaTranscriptAssetFilter.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * @package plugins.transcript
+ * @subpackage api.filters
+ */
+class KalturaTranscriptAssetFilter extends KalturaTranscriptAssetBaseFilter
+{	
+	/* (non-PHPdoc)
+	 * @see KalturaAssetFilter::getTypeListResponse()
+	 */
+	public function getTypeListResponse(KalturaFilterPager $pager, KalturaDetachedResponseProfile $responseProfile = null, array $types = null)
+	{
+		$types = KalturaPluginManager::getExtendedTypes(assetPeer::OM_CLASS, TranscriptPlugin::getAssetTypeCoreValue(TranscriptAssetType::TRANSCRIPT));
```

Comment:
@tan-tan-kanarek I removed the `$typeIn` property as it seems to not being used, the type is forced from `$types` and we must override it here, otherwise we would get both TRANSCRIPT & ATTACHMENT in the query.